linchpin/__init__.py

```diff
@@ -177,7 +177,7 @@ def destroy(ctx, pinfile, targets):
 
     \b
     TARGET ...
-    Destroy ONLY the listed target(s).
+    Destroy ONLY the listed target(s). If omitted, ALL targets will be destroyed.
```

Comment:
Great idea here! Maybe it should read 'If omitted, ALL targets in the appropriate PinFile will be destroyed.'